### PR TITLE
Add content on migrating to Choreo

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Add content on migrating to Choreo.

### CODE REVIEW

File: en/docs/migrate-to-choreo/api-cloud-vs-choreo-feature-comparison.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+| **Environment support**                  | **NO -** A separate organization (tenant) must be created for each environment (development, production, etc). | **YES -** Two environments (development and production)  are supported by default within an organization. Customers can create more environments depending on their requirements. For more details, see [Choreo environments](https://wso2.com/choreo/docs/develop/components/rest-api/#choreo-environments). |
+| **REST APIs**                            | **YES**                                                                                                       | **YES** <br>For more details, see [REST API proxy](https://wso2.com/choreo/docs/develop/components/api-proxy/).                                                                                                                                                |
+| **APIs for SOAP backends**               | **YES**                                                                                                        | Need to implement a REST API from scratch to front the SOAP backend using a preferred programming language.                                             |
```
Comment (isuruw) [inline en/docs/migrate-to-choreo/api-cloud-vs-choreo-feature-comparison.md]: Lets change this to "Can be done by implementing a Ballerina REST API"

File: en/docs/migrate-to-choreo/why-migrate-from-api-cloud-to-choreo.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+-  The Choreo Console (equivalent to the API Publisher in API Cloud) allows API developers to design, publish, and manage the lifecycle of an API.
+-  The API Developer Portal (equivalent to the API Store in API Cloud) allows consumers to consume APIs. 
+
+Overall, Choreo provides users with an improved user experience in API management.
+
+## End of life of WSO2 API Cloud
+WSO2 API Cloud was the WSO2 SaaS platform for API Management from 2015 onwards. Now, with the introduction of Choreo, the operations of WSO2 API Cloud comes to an end. The deadline for the end of life of API Cloud is December 31, 2023.
+
+## A futuristic digital innovation platform for API Cloud customers
```
Comment (isuruw) [inline en/docs/migrate-to-choreo/why-migrate-from-api-cloud-to-choreo.md]: The subject may not be suitable, the purpose of this section is to let customers know what are the options they have when they are asked to move away from API Cloud, what should be their future direction within WSO2.

File: en/docs/migrate-to-choreo/how-to-migrate-to-choreo.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# How to Migrate to Choreo?
+
+## Migration process
+
+-  It is not possible to automate the migration process due to several reasons. WSO2 API Cloud and Choreo are two distinct platforms with different architectures, making an automated process too complicated. Additionally, some API Cloud customers have customizations (such as custom domains and user stores) that require manual migration.
```
Comment (isuruw) [inline en/docs/migrate-to-choreo/how-to-migrate-to-choreo.md]: process "is" too complicated ?

File: en/docs/migrate-to-choreo/api-cloud-vs-choreo-feature-comparison.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/migrate-to-choreo/api-cloud-vs-choreo-feature-comparison.md]: Updated as suggested.

File: en/docs/migrate-to-choreo/why-migrate-from-api-cloud-to-choreo.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/migrate-to-choreo/why-migrate-from-api-cloud-to-choreo.md]: Updated to "Future of WSO2 API Cloud customers" to align with the original subheading in the gdoc.

File: en/docs/migrate-to-choreo/how-to-migrate-to-choreo.md (hunk 3 above)
Comment (Nashaath) [reply en/docs/migrate-to-choreo/how-to-migrate-to-choreo.md]: Rephrased the sentence to improve clarity.

**Review by isuruw — APPROVED** [review]
LGTM

**Pull request merged**